data/analytics/events.yml

```diff
@@ -76,6 +76,28 @@
         - name: external
         - name: method
         - name: link_domain
+
+- name: back link
+  description:
+  events:
+    - name: link click
+      description: When the back link is used, on pages like the email subscription journeys
```

Comment:
Is it worth using the `examples` value here to provide a link to an example?

Reply:
Yeah I wanted to but linking directly into the subscriptions would only work if someone was signed in. There's got to be other examples around the site but I can't find any right now. I'm hoping once the tracking is enabled we'll be able to put an actual example in here.

Reply:
No worries 👍